### ISSUE TITLE
The documentation page is lacking pictorial representation.

### ISSUE THREAD

**Issue by dossone1** [MicrosoftDocs/CommunityToolkit#187]
The documentation page lacks pictorial view of the behaviour of the tool being talked about.
This may sound like a minor issue and that any serious developer should have no issue with it but the framework is not just for serious developers. There are beginners who pictures does magic for. Its the picture that shifts their focus to seeing the feature at work. It amy surprise you to know that, there may be lots of people who will say this feature does not exist even though they may have visited the docs page.
Pictures of the features in action throws more light that detailed written documentations.

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: c6126a74-78aa-2c4f-ce92-2cda527ca9fd
* Version Independent ID: c6126a74-78aa-2c4f-ce92-2cda527ca9fd
* Content: [Popup - .NET MAUI Community Toolkit - .NET Community Toolkit](https://learn.microsoft.com/en-us/dotnet/communitytoolkit/maui/views/popup)
* Content Source: [docs/maui/views/popup.md](https://github.com/MicrosoftDocs/CommunityToolkit/blob/main/docs/maui/views/popup.md)
* Product: **dotnet-communitytoolkit**
* GitHub Login: @bijington
* Microsoft Alias: **bramin**

**Comment by bijington** [comment]
I completely agree that we should have images showing the result of a `Popup`. We must have missed adding some in for this concept.

**Comment by bijington** [comment]
This is a duplicate of #68 but happy to keep this open as a reminder
